README with DeviceAuthenticity API details

ALIIQBAL786 opened the pull request:
Updated the README to provide a detailed description of the DeviceAuthenticity API, including its functionalities and scope.

#### What type of PR is this?

Add one of the following kinds:
* documentation
* repository management


#### What this PR does / why we need it:
Adds README.md into repo.



#### Which issue(s) this PR fixes:

<!-- Automatically closes linked issue when PR is merged.
Usage: `Fixes #<issue number>`, or `Fixes (paste link of issue)`. -->

Fixes #

#### Special notes for reviewers:



#### Changelog input

```
 release-note

```

#### Additional documentation 

This section can be blank.



```
docs

```

ALIIQBAL786 committed:
Enhance README with DeviceAuthenticity API details

Updated the README to provide a detailed description of the DeviceAuthenticity API, including its functionalities and scope.

hdamker commented on README.md:
```diff
@@ -9,17 +9,25 @@
 # DeviceAuthenticity
 
-Sandbox API Repository to describe, develop, document, and test the DeviceAuthenticity Service API(s). The repository does not yet belong to a CAMARA Sub Project.
+The **DeviceAuthenticity API** enables applications to verify the authenticity and fraud status of mobile devices using their International Mobile Equipment Identity (IMEI).  
```
Maybe you want to consider to allow other device identifiers (as provided by DeviceIdentifier, e.g. a pairwise pseudonymes one), just to avoid that for this check the IMEI need to be exposed. 

```suggestion
The **DeviceAuthenticity API** enables applications to verify the authenticity and fraud status of mobile devices using their device identifier, e.g. the International Mobile Equipment Identity (IMEI).  
```

hdamker replied:
Please leave this line untouched and move this sentence into the scope. We don't explain the APIs in this line but have the scope section for that.

hdamker commented on README.md:
```diff
@@ -9,17 +9,25 @@
+  * Verify IMEI: Check if the IMEI is valid.
+  * Check device status: See if the device is lost, stolen, blacklisted, involved in fraud/non-payment, or restricted by regulations.
+  * Return clear status codes like ALLOWED, PROHIBITED, TRACKED, UNKNOWN.
+  * It also returns sub-statuses LOST, STOLEN, BLACKLISTED, BLOCKED, FRAUD, NON_PAYMENT, REGULATORY.
```
That's maybe already too detailed. You would need to change the scope if there will be changes during the API design.

hdamker commented on README.md:
```diff
@@ -9,17 +9,25 @@
-* The API(s) provide(s) the API consumer with the ability to:  
-  * §Short API scope description§ !! Add here a scope description according to the API proposal
+* The API provides the API consumer with the ability to:  
+  * Verify IMEI: Check if the IMEI is valid.
```
Verify IMEI is maybe better suited within DeviceIdentifier? Propose to discuss the scope within the the call together with DeviceIdentifier to avoid overlaps.

ALIIQBAL786 replied:
Yup, that would be great. Thank you for the suggestion.

ALIIQBAL786 committed:
Update README.md

Chnaging IMEI number to a generic term device identifier.

Co-authored-by: Herbert Damker <herbert.damker@telekom.de>

ALIIQBAL786 committed:
Revise DeviceAuthenticity API description in README

Updated the DeviceAuthenticity API section to clarify its purpose and capabilities.
